erica_app/erica/request_processing/requests_controller.py

```diff
@@ -222,3 +221,73 @@ def generate_json(self, pyeric_response: PyericResponse):
         response = super().generate_json(pyeric_response)
         response['address'] = get_address_from_xml(pyeric_response.server_response)
         return response
+
+
+class GetTaxOfficesRequestController:
```

Comment:
Yes, I was thinking about that do. I left it here because if you look at the abstraction layers from bottom to top (Eric -> routes) this is the abstraction layer where we normally do that.
That said, I see the following two places:
- Create an additional PyericControllor that can be used as if that would be an actual Eric functionality
- Add this to the script as this is the only place where we use it anyway

Reply:
I think a PyericController would be a reasonable place as that is the translation layer between ERiC and the Erica domain, right? But the script would also make sense.